CONTRIBUTING.md

```diff
@@ -8,6 +8,8 @@ Independent contributions (i.e., individual pull requests) from anyone other tha
 
 Meaning, that all independent contributors retain ownership of their contributions, albeit non-exclusively. In other words, your contributions belong equally to the Weird project as they do to you.
 
+Furthermore, as a consequence of our [Polyform Countdowm License Grant](https://polyformproject.org/licenses/countdown/1.0.0/), all code releases older than two years are solely licensed as Blue Oak Model License v1.0, i.e. no longer dual-licensed.
+
```

Comment:
The dual-licensing is a little confusing, because normally, in the Rust world, when something is dual licensed, it is dual licensed **at your option** between the two licenses.

In this case, we are dual licensing, but not giving you an option, because if we gave you an option you could just choose Blue Oak and use it for commercial purposes.

Maybe we add something like:

```suggestion

> **Note:** While the code is dual licensed, it is dual licensed under both licenses **without** an option between them, aka `Polyform-NonCommercial and Blue-Oak-1.0`. This is in contrast to other dual licenses such as `MIT or Apache-2.0`, which allows an option between the two licenses.
```

Reply:
The dual-licensing works the same way for us. The difference is that we are only dual-licensing *individual PRs* from contributors outside of the Weird team. We are not dual-licensing our codebase as a whole.

If someone wanted to reuse the code of a Weird contributor’s PR, they could indeed use that code under the terms of Blue Oak.

Licensing easily gets confusing, especially when going off the beaten path as we are. But I’d rather keep our legalese as brief as possible and address the confusion as it arises, rather than trying to preempt it with even more words.

Reply:
> The difference is that we are only dual-licensing individual PRs from contributors outside of the Weird team. We are not dual-licensing our codebase as a whole.

Oohh... That makes sense. I had missed that, even though that is indeed what it says.

OK, I'm good. :) 👍️